africanlii/templates/africanlii/set_language.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
```

Comment:
All of this will have to move to peachjam (rather than africanlii) at some point, so I recommend doing it from the start.